pipeline/backend/kubernetes/kubernetes.go

```diff
@@ -147,9 +152,21 @@ func (e *kube) Setup(ctx context.Context, conf *types.Config) error {
 
 // Start the pipeline step.
 func (e *kube) Exec(ctx context.Context, step *types.Step) error {
-	pod := Pod(e.config.Namespace, step)
+	labels := make(map[string]string)
```

Comment:
could you deserialize this on `configFromCliContext` instead on first exec

so we fail on agent start instead if a step should be exec

Reply:
Done